File: 06-06-2025/QuizupAPI/Models/QuizQuestion.cs
namespace QuizupAPI.Models
{
    public class QuizQuestion
    {
        public long Id { get; set; }
        public long QuizId { get; set; }
        public Quiz? Quiz { get; set; }
        public long QuestionId { get; set; }
        public Question? Question { get; set; }
    }
}

File: Quizup-Angular-Application/src/app/features/quiz/quiz-question/quiz-question.ts
import { Component, Input, OnInit } from '@angular/core';
import { QuestionModel } from '../models/question.model';
import { CommonModule } from '@angular/common';

@Component({
  selector: 'app-quiz-question',
  standalone: true,
  imports: [CommonModule],
  templateUrl: './quiz-question.html',
  styleUrl: './quiz-question.css',
})
export class QuizQuestion implements OnInit {
  @Input() question: QuestionModel | null = null;

  ngOnChanges() {
    console.log('QuizQuestion input updated:', this.question);
  }

  ngOnInit() {
    // Optional: use only for non-@Input setup
  }
}
